code/lab1-tdd-arabic-roman-converter/src/test/java/ru/unn/agile/ArabicRomanConverter/ArabicToRomanTest.java

```diff
@@ -0,0 +1,157 @@
+package ru.unn.agile.ArabicRomanConverter;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ArabicToRomanTest {
+
+    private String convertToRoman(final int arabic) {
+        ArabicToRomanConverter converter = new ArabicToRomanConverter();
+        return converter.convert(arabic);
+    }
+
+    @Test
+    public void canConvertToEmptyString() {
+        String romanNumber = convertToRoman(0);
+        assertEquals("", romanNumber);
+    }
+
+    @Test
+    public void canConvert1() {
+        String romanNumber = convertToRoman(1);
+        assertEquals("I", romanNumber);
+    }
+
+    @Test
+    public void canConvert2() {
+        String romanNumber = convertToRoman(2);
+        assertEquals("II", romanNumber);
+    }
+
+    @Test
+    public void canConvert3() {
+        String romanNumber = convertToRoman(3);
+        assertEquals("III", romanNumber);
+    }
+
+    @Test
+    public void canConvert5() {
```

Comment:
why tests are not in correct order? CanConvert3 -> CanConvert4 -> CanConvert5

Reply:
Done